README.md

```diff
@@ -1 +1,9 @@
-# SYN-IN064
\ No newline at end of file
+def is_prime(num):
```

Comment:
Please call the defined function to get the demonstration.
Your code should not be pushed to README file, push your code to .py(python) file